src/lib/eina/eina_debug_timer.c

```diff
@@ -95,17 +96,22 @@ _monitor(void *_data EINA_UNUSED)
    ret = epoll_ctl(epfd, EPOLL_CTL_ADD, event.data.fd, &event);
    if (ret) perror("epoll_ctl/add");
 #ifdef EINA_HAVE_PTHREAD_SETNAME
+/*
 # ifndef __linux__
    pthread_set_name_np
 # else
    pthread_setname_np
 # endif
      (pthread_self(), "Edbg-tim");
+*/
 #endif
+   eina_thread_name_set(eina_thread_self(), "Edbg-tim");
    while (1)
      {
         int timeout = -1; //in milliseconds
-        pthread_testcancel();
+        #ifndef _WIN32
+         pthread_testcancel();
+        #endif
```

Comment:
The case here is the same as in `pthread_getcpuclockid`?

Reply:
yes